app/lib/common/providers/notifiers/chat_notifiers.dart

```diff
@@ -51,7 +52,7 @@ class AsyncLatestMsgNotifier extends FamilyAsyncNotifier<RoomMessage?, String> {
   FutureOr<RoomMessage?> build(String arg) async {
     final roomId = arg;
     final client = await ref.watch(alwaysClientProvider.future);
-    _listener = client.subscribeStream('$roomId::latest_message');
+    _listener = client.subscribeRoomParamStream(roomId, 'latest_message');
```

Comment:
there is still a few magical keywords for these functions that are then understood via strums EnumString, that I'd like to also switch into typed enums on the dart-side, but I opted for doing that in a follow up as this PR is already quite large as it is...

Reply:
Yeah, it's good to have also these string be typed as enums also. But overall good refactor on making these more safe!